modules/moordyn/src/MoorDyn_C_Binding.f90

```diff
@@ -144,7 +144,7 @@ SUBROUTINE MD_C_Init(                                             &
    REAL(C_FLOAT)                                  , INTENT(IN   )   :: G_C
    REAL(C_FLOAT)                                  , INTENT(IN   )   :: RHO_C
    REAL(C_FLOAT)                                  , INTENT(IN   )   :: DEPTH_C
-   REAL(C_FLOAT)                                  , INTENT(IN   )   :: PtfmInit_C(6)
+   REAL(C_FLOAT)                                  , INTENT(IN   )   :: PtfmInit_C(6) ! TODO: make this more flexible, can we not have 6 DOF only coupling?
```

Comment:
At some point we need to allow multiple connection points.  At present it is a single node only through the interface.